docs/modules/device-lab-management/pages/android-devices/prepare-chinese-android-device.adoc

```diff
@@ -0,0 +1,63 @@
+= Prepare a Chinese Android device for hosting
+:navtitle: Prepare a Chinese Android device
```

Comment:
@Hikkigaya As this page is specific for Xiaomi devices only, can you change the title to reflect that?

Reply:
Also this guide only provide instructions to enable debugging (developer mode) for Xiaomi devices, so the title should be "Enable Developer mode".

Reply:
Changed the title to "Enable Developer mode for a Chinese Android device"